cmd/gcp-controller-manager/node_annotator.go

```diff
@@ -271,17 +276,70 @@ func extractKubeLabels(instance *compute.Instance) (map[string]string, error) {
 		return make(map[string]string), nil
 	}
 
+	parsedLabels, err := parseLabels(*kubeLabels)
+	if err != nil {
+		return nil, fmt.Errorf("instance %q had %s", instance.SelfLink, err.Error())
+	}
+	return parsedLabels, nil
+}
+
+func extractLastAppliedLabels(node *core.Node) (map[string]string, error) {
```

Comment:
Return error is always nil?

Reply:
Yes we decided to suppress error (only log it) when failed to parse last applied label annotation. Removed return arg 'err'.